### PR TITLE
WDI: Add more StatVars for int'l place pages

### PR DESCRIPTION
**Pull request by tjann**

### CODE REVIEW

File: scripts/world_bank/output/WorldBank.tmcf [hunk 1]
```diff
@@ -0,0 +1,80 @@
+
```
Comment (shifucun) [inline scripts/world_bank/output/WorldBank.tmcf:1]: nit: nice to remove extra first line

File: scripts/world_bank/worldbank.py [hunk 2]
```diff
@@ -75,52 +83,65 @@ def read_worldbank(iso3166alpha3):
...
-    df = df.stack()
-    df.index = df.index.rename('year', level=4)
-    df.name = "Value"
-    df = df.reset_index()
-
-    # Convert to numeric and drop empty values.
-    df['Value'] = pd.to_numeric(df['Value'])
-    df = df.dropna()
+    if fetchFromSource:
+        print(f"Downloading {iso3166alpha3}")
```
Comment (shifucun) [inline scripts/world_bank/worldbank.py:87]: nit: it's better to use logging.info() everywhere

File: scripts/world_bank/worldbank.py [hunk 3]
```diff
@@ -75,52 +83,65 @@ def read_worldbank(iso3166alpha3):
...
+        country_zip = ("http://api.worldbank.org/v2/en/country/" +
+                       iso3166alpha3 + "?downloadformat=csv")
+        r = requests.get(country_zip)
```
Comment (shifucun) [inline scripts/world_bank/worldbank.py:90]: if this is in general needed, can use: https://pypi.org/project/retry/

File: scripts/world_bank/worldbank.py [hunk 4]
```diff
@@ -139,15 +160,16 @@ def row_to_constraints(row):
...
     # yapf: disable
     # Input all required statistical variable fields.
-    new_stat_var = (TEMPLATE_STAT_VAR
-        .replace("{INDICATOR}", row['IndicatorCode'].replace(".", "_"))
-        .replace("{NAME}", row['IndicatorName'])
-        .replace("{DESCRIPTION}", row['SourceNote'])
-        .replace("{measuredProperty}", row['measuredProp'])
-        .replace("{CONSTRAINTS}", row_to_constraints(row))
-    )
+    new_stat_var = (TEMPLATE_STAT_VAR.replace(
```
Comment (shifucun) [inline scripts/world_bank/worldbank.py:166]: might think how to do this, its not quite readable now...

File: scripts/world_bank/output/WorldBank.tmcf (hunk 1 above)
Comment (tjann) [reply scripts/world_bank/output/WorldBank.tmcf:1]: Done, but shited the extra line to the end. I'm not sure that's any better.

File: scripts/world_bank/worldbank.py (hunk 2 above)
Comment (tjann) [reply scripts/world_bank/worldbank.py:87]: Done

File: scripts/world_bank/worldbank.py (hunk 4 above)
Comment (tjann) [reply scripts/world_bank/worldbank.py:166]: Yeah I agree. See my comment above

File: scripts/world_bank/worldbank.py (hunk 3 above)
Comment (tjann) [reply scripts/world_bank/worldbank.py:90]: Nice, thanks!

**Review by shifucun — APPROVED** [review]

**Pull request merged**